task/jefe/src/main.rs

```diff
@@ -252,6 +253,20 @@ impl idol_runtime::NotificationHandler for ServerImpl<'_> {
                 }
             }
         }
+        // Check whether all 'exit' tasks have 'Exited'
+        let mut done: bool = true;
+        for (i, exits) in TASK_EXITS.iter().enumerate() {
+            if *exits
+                && (kipc::read_task_status(i)
+                    != abi::TaskState::Healthy(abi::SchedState::Exited))
+            {
+                done = false;
+                break;
+            }
+        }
+        if done {
+            kipc::system_restart();
```

Comment:
Why system restart? I would think the default would just be power off.

Reply:
Agreed, although currently system_restart is the only kipc that we have available.  I will make the 'done' behaviour selectable via a feature as you suggested earlier.